cmd/adapter/api/api.go

```diff
@@ -357,6 +365,23 @@ func toGravityLogEntries(evs []*api.LogEvent) ([]string, error) {
 	return entries, nil
 }
 
+func parseCSVIntoMap(csv string) (results map[string]string) {
+
+	// iterate over the elements in the form `elem1, elem2, elem3, ...`
+	for _, elem := range strings.Split(csv, ",") {
+
+		// each element should be in the form `key=value`
+		elemSplit := strings.Split(elem, "=")
+		if len(elemSplit) == 2 {
+			results[elemSplit[0]] = elemSplit[1]
+		} else {
+			// one of the element was not in the form `key=value`
```

Comment:
Is it not valid to have just a key as a tag?

Reply:
Added case where there's a an empty tag with just the key.

Thanks!